analyses/new-feature-template.qmd

```diff
@@ -0,0 +1,1564 @@
+---
+title: "New Feature Template"
+subtitle: "Run ID: `r params$added_feature`: `r params$description`"
+date: "`r Sys.Date()`"
+author: "Cook County Assessor's Office Data Department"
+execute:
+  echo: false
+  warning: false
+  asis: true
+format:
+  html:
+    embed-resources: true
+    toc: true
+    toc_float: true
+    fig-align: center
+    fontsize: 12pt
+params:
+  run_id: "2024-07-03-charming-boni"
+  run_id_year: "2024"
+  comparison_run_id: "2024-07-09-cool-takuya"
+  comparison_run_id_year: "2024"
+  added_feature: "prox_nearest_new_construction_dist_ft" 
+  added_feature_shap: "prox_nearest_new_construction_dist_ft_shap"
+  description: "A distance in feet to the nearest new construction"
+  min_range: 5
+  max_range: 95
+  type: "continuous"
```

Comment:
It's a flexible parameter for if we expect improvement based on a specific range. For example, distance to new construction would likely be most indicative between 5 and 95 feet (un-coded as feet).

Reply:
Sorry, should've clarified the ask here: just add documentation/comments on these so people know their purpose (or rename them so their purpose is more obvious).

Reply:
Since we got rid of the manual ratio stats calculation, we don't use the range anymore. The only other thing I could think of doing it for is a scatterplot, but we can also just drop it.